errors/OrphanTypeDeclaration.md

```diff
@@ -5,17 +5,23 @@
 ```purescript
 module ShortFailingExample where
 
+addTwo :: Number -> Number -> Number
+addToo a b = a + b
 ...
 ```
 
 ## Cause
 
-Explain why a user might see this error.
+Purescript requires that the type declaration for a function or value (i.e. the line with the `::` in it)
+be followed immediately by the definition of that function or value (i.e. the line with the `=` in it). The
+compiler has found a declaration for which a definition did not follow.
 
-## Fix
-
-- Suggest possible solutions.
+This is often caused by typographical issues.  Check to see that the declaration and the
```

Comment:
Just a suggestion; while “typographical issues” is correct here, I think there’s a risk of not getting the point across (I suspect many people don’t know that this is what “typo” is short for).
```suggestion
This is often caused by spelling mistakes.  Check to see that the declaration and the
```

Reply:
I almost said this in my review...

Reply:
I thought about this.  This error also shows up with indentation issues which are not spelling errors technically speaking, but I have no problem if the community wants to use simpler language.

Reply:
I missed the error myself when reading the above example.
What about:
> This is often caused by spelling mistakes ("Two" vs "Too" in this case).